src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1386,4 +1386,7 @@ From there, simply click on a Markdown file or SVG icon in the File Explorer and
     <data name="ColorPicker_ColorFormat_ToggleSwitch.AutomationProperties.Name" xml:space="preserve">
     <value>Show format in editor</value>
   </data>
+  <data name="PowerLauncher_SearchPlugins.PlaceholderText" xml:space="preserve">
+    <value>Search plugins</value>
```

Comment:
```suggestion
    <value>Search plugins ...</value>
```

Reply:
Maybee better:
```suggestion
    <value>Search plugin's names ...</value>
```

Reply:
![image](https://user-images.githubusercontent.com/25966642/128064837-6c115d4a-73e2-4852-908d-253a9e9b45b4.png)

Wondering what to use since Windows 10 UI uses "find a setting" and "search this list".

Reply:
Search this list sounds good to me

Reply:
Windows 11 uses this labels:
- Settings search: "Find a setting"
- Uninstall apps list: "Search apps"
- Default apps > Type list: "Enter a file type or link type"
- Default apps > application list: "Search apps"
- Default apps > By file type: "Search for a file type"
- Default apps > By link type: "Search for a link type"
- Optional feature > History: "Search history"
- Optional feature > Installed features: "Search" (Might be a ux bug that the word `features` is missing.

**=> Based on the list above I suggest the phrase "Search plugins" like wevhave atm.**